protocol/daemons/liquidation/client/grpc_helper.go

```diff
@@ -190,6 +262,20 @@ func (c *Client) SendLiquidatableSubaccountIds(
 	return nil
 }
 
+// nolint:unused
+func newContextWithQueryBlockHeight(
```

Comment:
will call this outside of these grpc queries in the main daemon loop. unused for now so silencing lint

Reply:
can do in future pr so you don't have to wait for CI to run again

Reply:
yeah will do in a future PR